custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -728,6 +728,7 @@ def _prepare_docker_command(self, options, run_mode, raw_arguments):
         return docker_cmd
 
     def _run_inside_docker(self, options, run_mode, raw_arguments):
+        self._check_artifacts_and_get_run_language()
```

Comment:
Before we needed language only for scoring.
Now we need it to decide which deps to install: Rython or R